### ISSUE TITLE
differential-dataflow doesn't compile for compiler tests

### ISSUE THREAD

**Issue by remysucre** [ryzhyk/differential-datalog#90]
`stack test` log attached. Might be related to #84 - might be fixed if one tracks down which git version of timely & abomonation worked, and track that in `rust/differential_datalog/Cargo.toml`

**Comment by ryzhyk** [comment]
Yes, there was no way to do this until recently, since differential used to include the master branch of timely as a dependency. As of yesterday, timely and differential started using a more disciplined version tracking scheme based on git tags. I will soon create a PR that takes advantage of this.

**Comment by ryzhyk** [comment]
This should be fixed in the latest master. Could you confirm that you are now able to run tests?

**Comment by remysucre** [comment]
Yes I'm able to run the tests now
